### DISCUSSION TITLE
Caching Server Side Props

### DISCUSSION THREAD

**Discussion by shadowtime2000** [vercel/next.js#19202] [Help] [answered]
I [have a blog site](https://github.com/shadowtime2000/dev-blog) which just lists all my posts on [DEV](https://dev.to). If there are too many requests, I get a 429 error code from the DEV api and that is shown to the user. What would be a simple/easy way to implement a cache on `getServerSideProps`?

**Comment by ljosberinn** [comment]
why is it server rendered? sounds as static to me as can be

**Reply by shadowtime2000** [reply]
It is rendered on the server side because I don't want to have to trigger a rebuild every time I create a new DEV post and I want to avoid a large cumulative layout shift.

**Reply by ljosberinn** [reply]
🤷 how often do you post anyways, doesn't warrant rendering SSR at all

**Reply by shadowtime2000** [reply]
I also want it to list statistics such as reactions view and comments. DEV allows you to edit posts too, which  I do, and I don't want to have to trigger a rebuild whenever I edit a post.

**Comment by rafaelalmeidatk** [comment, marked as answer]
This sounds like a good scenario to use [Incremental Static Regeneration](https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration). You can set a number of seconds that it will take before the page is revalidated, which is basically the same as a cached `gSSP` response

**Reply by shadowtime2000** [reply]
Thanks! I didn't know you could do that.
